modules/release-notes/pages/relnotes.adoc

```diff
@@ -27,6 +27,19 @@ This release adds support for the following platforms:
 
 See xref:install:install-platforms.adoc[Supported Platforms] for the complete list of supported platforms.
 
+[#deprecated-features-and-platforms-710]
+== Deprecated Features and Platforms
+
+=== Deprecated and Removed Platforms
+
+The following platforms are deprecated and will be removed in a future release:
```

Comment:
In Neo..  
CentOS7 = Deprecated
CentOS8 = Removed
Debian9 = Removed
RHEL 7 = Deprecated
Ubuntu 18 = Deprecated
Microsoft Windows Server 2016 = Removed
Microsoft Windows Server 2016 (64-bit, DataCenter Edition) = Removed